src/System.CodeDom/src/System/CodeDom/Compiler/CompilerInfo.cs

```diff
@@ -103,7 +103,7 @@ internal CompilerInfo(CompilerParameters compilerParams, string codeDomProviderT
         public override bool Equals(object o)
         {
             CompilerInfo other = o as CompilerInfo;
-            if (o == null)
+            if (other == null)
```

Comment:
In general, we consider src/tests to be a unit, such that we ask contributors changing src to ensure that the tests for the thing being changed are appropriate.  We can skip doing so this time, but for any future contributions please do so.  Thanks for the contribution.